templates/wiqa/templates.yaml

```diff
@@ -0,0 +1,193 @@
+dataset: wiqa
+templates:
+  1bc8d95b-0a50-49f4-a46b-bd752929926d: !Template
+    id: 1bc8d95b-0a50-49f4-a46b-bd752929926d
+    jinja: '{% for k in range(1, question_para_step | length) %}
+
+      - {{question_para_step[k]}}
+
+      {% endfor %}
```

Comment:
```suggestion
    jinja: '- \n- {{ question_para_step | join("\n- ") }}
```

Reply:
same for `remove_first_step_bis`

Reply:
and `remove_last_step` and `remove_last_step_bis`

Reply:
Those were actually on purpose:
- for `remove_first_step` and `remove_first_step_bis`, the range starts at 1 to remove the first step. Not sure whether there's an easy way to using slicing with jinja
- for `remove_last_step` and `remove_last_step_bis`, the range stops at length-1 to remove the last step. Again not sure whether we can do the slicing directly with jinja

Reply:
Actually, my bad, slicing is working. Will update